ethereum/eth/src/test/java/org/hyperledger/besu/ethereum/eth/transactions/PendingTransactionsTest.java

```diff
@@ -619,4 +619,26 @@ public void shouldNotIncrementAddedCounterWhenLocalTransactionAlreadyPresent() {
     assertThat(metricsSystem.getCounterValue(ADDED_COUNTER, LOCAL)).isEqualTo(0);
     assertThat(metricsSystem.getCounterValue(ADDED_COUNTER, REMOTE)).isEqualTo(1);
   }
+
+  @Test
+  public void assertThatCorrectNonceIsReturned() {
+    addLocalTransactions(1, 2, 4, 5);
```

Comment:
Can we add an assert for "1" prior to adding any transactions?

Reply:
Or whatever the behavior _should_ be when we have no TXes for a particular address

Reply:
done. the expected behaviour is to return an empty Optional